config/prow/monitoring/mixins/vendor/github.com/prometheus/node_exporter/docs/node-mixin/Makefile

```diff
@@ -0,0 +1,32 @@
+JSONNET_FMT := jsonnetfmt -n 2 --max-blank-lines 2 --string-style s --comment-style s
+
+all: fmt node_alerts.yaml node_rules.yaml dashboards_out lint
+
+fmt:
+	find . -name 'vendor' -prune -o -name '*.libsonnet' -print -o -name '*.jsonnet' -print | \
+		xargs -n 1 -- $(JSONNET_FMT) -i
+
+node_alerts.yaml: mixin.libsonnet config.libsonnet $(wildcard alerts/*)
+	jsonnet -S alerts.jsonnet > $@
+
+node_rules.yaml: mixin.libsonnet config.libsonnet $(wildcard rules/*)
+	jsonnet -S rules.jsonnet > $@
+
+dashboards_out: mixin.libsonnet config.libsonnet $(wildcard dashboards/*)
+	@mkdir -p dashboards_out
+	jsonnet -J ../../../../../../vendor -J lib -m dashboards_out dashboards.jsonnet
```

Comment:
Only change from upstream is this line: we want to include our `config/prow/monitoring/mixins/vendor` vendor folder here.
It is a bit ugly :laughing: